api/src/pcapi/core/providers/repository.py

```diff
@@ -134,11 +137,40 @@ def get_ems_cinema_details(cinema_id: str) -> models.EMSCinemaDetails:
     return cinema_details
 
 
-def set_ems_cinema_sync_version(venue_provider: models.VenueProvider, version: int) -> None:
-    ems_cinema_pivot = get_cinema_provider_pivot_for_venue(venue_provider.venue)
-    if ems_cinema_pivot:
-        ems_cinema_details = get_ems_cinema_details(ems_cinema_pivot.idAtProvider)
-        ems_cinema_details.lastVersion = version
+def bump_ems_sync_version(version: int, venue_provider_to_sync: Iterable[int]) -> None:
+    """
+    Storing the version point (timestamp) from which we are up to update
+    """
+    ids: list[Sequence[int]] = (
+        models.EMSCinemaDetails.query.join(
+            models.CinemaProviderPivot, models.EMSCinemaDetails.cinemaProviderPivotId == models.CinemaProviderPivot.id
+        )
```

Comment:
(détail / style) On devrait pouvoir indiquer la relation en premier et seul argument de `join()`. Ça allège un peu : 

```suggestion
        models.EMSCinemaDetails.query.join(models.EMSCinemaDetails.cinemaProviderPivot)
```